frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -663,6 +663,16 @@ function sorted_names_from(subs) {
 }
 
 test("initialize", ({override, mock_template}) => {
+    // TODO: where's the right place to put this codeblock to share accross files?
+    // Or is it better to just copy it per file?
```

Comment:
This gets repeated a lot and would be nice to have in some test util file somewhere

Reply:
I think it's actually better to leave as this, since it's only 4 instances and there isn't much precedent for test lib files

Reply:
I can't see the context for this diff anymore, but in general it's reasonable to add common code to library files if it feels more readable. (Sometimes, just repeating ourselves is actually more readable; certainly that's going to be the case for things like import statements, where you'd like `grep` to help you find what modules import your module).